src/components/ProjectSearch/TextSearch.js

```diff
@@ -83,6 +99,12 @@ export default class TextSearch extends Component {
     }
   }
 
+  buildSummaryMsg(updateCount, resultCount) {
+    return !updateCount
+      ? ""
+      : L10N.getFormatStr("sourceSearch.resultsSummary1", resultCount);
```

Comment:
sorry... i think i'm missing the point here:

* will this only show a value if updateCount is true? when is it false?
* is `updateCount` representing that we have searched? can we do this another way? like looking at whether there is a current search query string? or adding a boolean to the text-search reducer? 

what is this trying to do again? :)

Reply:
Not quite sure I'm getting the direction. 

The idea behind it is to show a count only after the enter key has been pressed (which will cause `updateCount = true`). Since the code automatically adjusts the `resultCount` disregarding of whether enter was pressed or not it makes the variable unreliable as means of determining whether or not to show something - we have a 0 results state while searching and a 0 results state that should not be shown because the enter key has not yet been pressed.